packages/spor-bottom-drawer-react-native/src/DrawerHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Heading } from "@vygruppen/spor-typography-react-native";
+
+type DrawerHeaderProps = {
+  children: string;
+  size: "sm" | "lg";
+  textAlign?: "left" | "center";
+  leftIcon?: JSX.Element;
+  rightIcon?: JSX.Element;
```

Comment:
Disse to er ikke i bruk per i dag – men det burde de kanskje være?

Reply:
Tok det med fordi jeg så det var støtte for det i appen, men er ikke noen figmaskisser på det så kanskje det skal droppes?

Reply:
Droppet det